.gitpod.yml

```
@@ -0,0 +1,10 @@
+image:
+  file: .docker/Dockerfile
+
+vscode:
+  extensions:
+    - jroesch.lean
+
+tasks:
+  - init: leanpkg configure && leanproject get-cache --fallback=download-all
+    command: . /home/gitpod/.profile
```

```suggestion
    command: . /home/gitpod/.profile

```
Is this a missing line break?  :smile:

---

I borrowed the gitpod file as is from mathlib3